overview.md

```diff
@@ -142,7 +144,8 @@ people are gaming.
 Second, since SourceCred is itself open-source, anyone can propose new heuristics to
 mitigate gaming. For example, the community could develop a machine learning model
 to distinguish between threads that are constructive and threads that are flame wars,
-and then greatly reduce the weight on the flame wars.
```

Comment:
Again, I think the added concept would be better expressed as a followon sentence. That way this sentence makes one point clearly, and the next sentence follows with another point clearly